src/Adapter/Customer/QueryHandler/GetCustomerForViewingHandler.php

```diff
@@ -209,7 +209,7 @@ private function getPersonalInformation(Customer $customer)
             $registrationDate,
             $lastUpdateDate,
             $lastVisitDate,
-            $this->getCustomerRankBySales($customer->id),
+            (string) $this->getCustomerRankBySales($customer->id),
```

Comment:
this method is casted to `int` just to be later casted to `string`, could you tell me what is the point of that?

Reply:
Fixed & BC Break explained.